### DISCUSSION TITLE
Как использовать unstable компоненты TextTooltip или RichTooltip

### DISCUSSION THREAD

**Discussion by painkkiller** [VKCOM/VKUI#3877] [Q&A] [answered]
Вероятно вопрос глупый и я просто упустил какую то деталь, но мне нужно использовать тултипы, и их внешний вид мягко говоря не соответствует заявленному в документации. 
Импортирую компоненты:
```
import { RichTooltip, TextTooltip } from "@vkontakte/vkui/unstable";
```
Если надо импортнуть какую то css то в доках про это не сказано, а в папке unstable я ничего подходящего не нашел.

Оборачиваю в них спаны. Суть компонента -  происходит проверка грамматики введенного текста. И по результатам ответа от сервера надо подсветить слова с ошибками, и в тультипе показать правильный вариант. Но независимо от того, что передано в appearance и назависимо от того что я пихаю в content ричтултипа, выглядят они одинаково:

![ReachTooltip](https://user-images.githubusercontent.com/5506725/209174841-fb0e659d-2b6b-4c9d-b68f-e23b5b61578b.png)
![ReachTooltip2](https://user-images.githubusercontent.com/5506725/209174850-be75f895-c5a3-4d12-a8e6-dbd4681096a7.png)

**Comment by SevereCloud** [comment, marked as answer]
Привет, документация вот [тут](https://vkcom.github.io/VKUI/4.40.0/#/Unstable)

```js
import "@vkontakte/vkui/dist/unstable.css";
```

**UPD**

В **v5** подход поменялся, см. https://vkcom.github.io/VKUI/5.5.0/#/Unstable

**Comment by painkkiller** [comment]
Спасибо, так и знал глупая ошибка. Извиняюсь за беспокойство.

**Reply by inomdzhon** [reply]
Бывает)
Спасибо за развёрнутый вопрос 🙏
